applications/MeshMovingApplication/custom_utilities/fixed_mesh_ale_utilities.cpp

```diff
@@ -468,19 +471,19 @@ namespace Kratos
 
     void FixedMeshALEUtilities::RevertMeshDisplacementFixity()
     {
-        #pragma omp parallel for
-        for (int i_node = 0; i_node < static_cast<int>(mrVirtualModelPart.NumberOfNodes()); ++i_node) {
-            auto it_node = mrVirtualModelPart.NodesBegin() + i_node;
-            if (it_node->IsFixed(MESH_DISPLACEMENT_X)) {
-                it_node->Free(MESH_DISPLACEMENT_X);
+        block_for_each( mrVirtualModelPart.Nodes(),
+        []( Node<3>& node )
```

Comment:
```suggestion
        []( Node<3>& r_node )
```

@matekelemen this is how we name references
Did I show you the [Style Guide](https://github.com/KratosMultiphysics/Kratos/wiki/Style-Guide) already?

can you please do those minor changes?
Thx

Reply:
I haven't checked the style guide before, sorry. I renamed the reference arguments and fixed a small lambda capture issue, please take a look at it and let me know if I missed anything.